contracts/ReversibleICO.sol

```diff
@@ -337,58 +336,80 @@ contract ReversibleICO is IERC777Recipient {
 
     }
 
+
     /**
-    @notice Commits a participant's ETH.
+    @notice External wrapper for commit() so that a participant can call it directly.
     */
     function commit()
     public
     payable
+    {
+        // Reject contributions lower than the minimum amount
+        require(msg.value >= minContribution, "Value sent is less than minimum contribution.");
```

Comment:
`cancel()` needs to be also here so that people with NO custom interfaces or wallets can cancel by simply sending below the minContribution.

So that there are two ways: calling `cancel()` OR sending 0 value or below minContribution to the smart contract.

I think we had a misunderstanding here.

Reply:
I think you got confused, this it commit() not fallback() ;)